feature-libs/cart/saved-cart/core/services/saved-cart-page-meta.resolver.ts

```diff
@@ -0,0 +1,112 @@
+import { Injectable } from '@angular/core';
+import {
+  BasePageMetaResolver,
+  BreadcrumbMeta,
+  PageBreadcrumbResolver,
+  PageMetaResolver,
+  PageTitleResolver,
+  PageType,
+  RoutingService,
+  SemanticPathService,
+  TranslationService,
+} from '@spartacus/core';
+import { combineLatest, defer, Observable, of } from 'rxjs';
+import {
+  distinctUntilChanged,
+  map,
+  shareReplay,
+  switchMap,
+} from 'rxjs/operators';
+
+/**
+ * Resolves the page data for Organization Pages.
+ *
+ * Breadcrumbs are built in this implementation only.
+ *
+ * @property {string} SAVED_CART_SEMANTIC_ROUTE the default root path for saved cart pages.
+ * @property {string} SAVED_CART_TRANSLATION_KEY the default i18n key for the saved cart breadcrumb label.
+ * @property {string} SAVED_CART_HOME_PATH the default path for Saved Cart home poge.
+ */
+@Injectable({
+  providedIn: 'root',
+})
+export class SavedCartPageMetaResolver
+  extends PageMetaResolver
+  implements PageBreadcrumbResolver, PageTitleResolver {
+  /**
+   * Translation key for the breadcrumb of Organization home page
```

Comment:
Saved cart 👍 

but I guess similar to organization, we can follow it's practices and make it like cart.breadcrumb